examples/grafana-integration/README.md

```diff
@@ -0,0 +1,56 @@
+# Hot R.O.D. - Rides on Demand  - Grafana integration
+
+This is the Hot R.O.D. demo application that consists of the same components as the `examples/hotrod/`, only Grafana, Loki and Prometheus integration is added to this setup, so you can correlate logs, metrics and traces in one application.
+
+## Running via `docker-compose`
+
+### Prerequisites
+
+* Clone the Jaeger repository `git clone https://github.com/jaegertracing/jaeger.git`, then `cd examples/grafana-integration`
+
+* All services will log to Loki via the loki docker logging plugin.
+To install run this on the host:
```

Comment:
```suggestion
* Install the Loki logging plugin for Docker:
```
If the plugin has its own page/README, I would link `Loki logging plugin` to it.

Reply:
Good point, linked.